traveler-swift-core/TravelerKit/Extensions/DateFormatter+PassengerKit.swift

```diff
@@ -44,6 +44,7 @@ extension DateFormatter {
     public static var dateOnlyFormatter: DateFormatter = {
         let formatter = DateFormatter()
         formatter.dateFormat = "yyyy-MM-dd"
+        formatter.timeZone = TimeZone(abbreviation: "UTC")
```

Comment:
This formatter is used a lot of different places. Sometimes to convert dates to strings and sometimes strings to dates. Will this cause an issue?
Also why would dates be displayed incorrectly in the absence of this? There is no notion of time in this formatter and it is purely date. The only thing I can imagine is that we may convert a Date which has time into a date only format and because of time different the days may change, even then I think we would want the day the user had intended and not convert it